spec/comments_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  user = User.create(name: 'Alphonce', photo: 'https:/photo.com', bio: 'Teacher', posts_counter: 5)
+  post1 = Post.create(author: user, title: 'Chinua', commentscounter: 10, likescounter: 7)
+
+  subject { Comment.new(post: post1, author: user, text: 'I am writing my first comment!') }
+  before { subject.save }
+
```

Comment:
Please add a test for `#update_comments_counter` in Comment model specs as any custom methods in the models should have corresponding unit tests.